collectors/scm/gitlab/src/main/java/com/capitalone/dashboard/gitlab/GitlabRequestsResponseMapper.java

```diff
@@ -100,8 +101,25 @@ private GitRequest map(String repoUrl, String branch, GitlabRequest gitlabReques
 		if (merged != null && merged.length() >= 10) {
 			long mergedTimestamp = new DateTime(merged).getMillis();
 			request.setScmCommitTimestamp(mergedTimestamp);
-			request.setResolutiontime((mergedTimestamp - createdTimestamp) / (24 * 3600000));
+			request.setResolutiontime((mergedTimestamp - createdTimestamp));
 			request.setMergedAt(new DateTime(merged).getMillis());
+			// get merge request changes' detail info
+            HttpHeaders headers = new HttpHeaders();
```

Comment:
Since this is done 4 times now, can you pull the header code out into a get headers method?

Reply:
Fixed by 6259c258e853407692bc5b7fda5d9ffb2e2e6002

Reply:
Thanks @zhenweiding . @Sbrenthughes Can you provide your approval so that we can get this PR merged?